deployment-detector/README.md

```diff
@@ -1,27 +1,26 @@
-# Challenge-1
+# Nethermind Forta Bots Deployer Detector
 
 ## Description
+This  monitors bot deployments from the Nethermind deployer address (0x88dC3a2284FA62e0027d6D6B1fCfDd2141a143b8)
 
-This agent detects forta bots that are deployed on behalf of nethermind.
-
-## Supported Chains
-
+## Supported ChainsS
 - Polygon
 
-
 ## Alerts
-
-Describe each of the type of alerts fired by this agent
-
-- FORTA-1
-  - Fired when Nethermind contract at :"0x88dc3a2284fa62e0027d6d6b1fcfdd2141a143b8" deploys an agent
+- NETH-BOT-DEPLOYER
+  - Fired when Nethermind contract at: [0x88dc3a2284fa62e0027d6d6b1fcfdd2141a143b8](https://polygonscan.com/tx/0x88dc3a2284fa62e0027d6d6b1fcfdd2141a143b8) deploys a bot
```

Comment:
```suggestion
  - Fired when Nethermind account at: [0x88dc3a2284fa62e0027d6d6b1fcfdd2141a143b8](https://polygonscan.com/tx/0x88dc3a2284fa62e0027d6d6b1fcfdd2141a143b8) deploys a bot
```
The address is actually an EOA rather than a contract.